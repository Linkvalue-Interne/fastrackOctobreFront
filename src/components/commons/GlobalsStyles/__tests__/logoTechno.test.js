import React from 'react';
import renderer from 'react-test-renderer';
import serializer from 'jest-emotion';

import Img from '../logoTechno';

expect.addSnapshotSerializer(serializer);

describe('Logo', () => {
  it('should renders with correct styles', () => {
    const props = {
      height: '10px',
      width: '10px',
      marginBottom: '3px',
      margin: '#fff',
    };

    const tree = renderer.create(<Img {...props} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
