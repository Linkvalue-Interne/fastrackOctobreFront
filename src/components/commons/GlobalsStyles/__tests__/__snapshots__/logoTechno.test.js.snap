// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should renders with correct styles 1`] = `
.emotion-0 {
  width: 10px;
  margin-bottom: 3px;
  margin: #fff;
}

<img
  className="emotion-0"
  height="10px"
  width="10px"
/>
`;
